server/index.ts

```diff
@@ -48,20 +51,44 @@ app.use((req: Request, res: Response, next: NextFunction) => {
   next();
 });
 // get the logged in user
-app.get('/user', requiresAuth(), (req: any, res: any) => {
-  res.send(req.oidc.user);
+app.get('/', (req: Request, res: Response) => {
+  res.send(req.oidc.isAuthenticated() ? 'Logged in' : 'Logged out');
 });
 
+declare global {
+  namespace Express {
+    interface Request {
+      user: User;
+    }
+  }
+}
+
 // get logged in user profile
 // * Must be /profile for Auth0 to work
-app.get('/profile', requiresAuth(), (req: any, res: any) => {
-  const user = req.oidc.user;
+app.get('/profile', requiresAuth(), (req: Request, res: Response) => {
+  const jason = JSON.stringify(req.oidc.user, null, 2);
+  const user = JSON.parse(jason);
+
   const currentUser = {
-    name: user.name,
-    email: user.email,
+    userId: user.sub,
```

Comment:
I dont see a change to the db schema to reflect this new column, is this causing issues?